src/cmd-sign

```diff
@@ -325,15 +324,19 @@ def robosign_oci(args, s3, build, gpgkey):
         for tag in image['tags']:
             identity = f"{image['image']}:{tag}"
             identities.setdefault(identity, []).append(image['digest'])
+            print(f"Signing for {identity} with digest {image['digest']} ({arch})")
+            if manifest_list_digest is None:
+                manifest_list_digest = image.get('manifest-list-digest')
 
     # For the manifest list digest, reuse the tags from the x86_64 build. As
     # mentioned above, it's the same tags on all arches.
-    if args.manifest_list_digest:
+    if manifest_list_digest:
```

Comment:
I'm trying to come up with a situation where `manifest_list_digest` would ever be a false value here. Is the if condition needed or should this block just be unconditional?